resilience4j-spring-boot2/src/main/java/io/github/resilience4j/bulkhead/autoconfigure/BulkheadAutoConfiguration.java

```diff
@@ -38,7 +39,7 @@
 @Configuration
 @ConditionalOnClass(Bulkhead.class)
 @EnableConfigurationProperties(BulkheadProperties.class)
-@Import(BulkheadConfigurationOnMissingBean.class)
+@Import({BulkheadConfigurationOnMissingBean.class,  RecoveryConfiguration.class})
```

Comment:
Here you need to follow the same pattern of condition of missing beans to allow user to override the recovery configuration if he needs so .

Reply:
I wrote `RecoveryConfigurationOnMissingBean` and added `Import` to each configuration it needed